iocore/net/quic/QUICStreamManager.cc

```diff
@@ -45,6 +45,12 @@ QUICStreamManager::interests()
   };
 }
 
+void
+QUICStreamManager::close_all_streams(int event)
+{
+  // TODO: close all stream, send the sm something: error, eos ??
```

Comment:
Let's leave this part as TODO on this PR. Stream state management need to be cleanup first.